### ISSUE TITLE
~username URLs cause fatal crashes

### ISSUE THREAD

**Issue by GoogleCodeExporter** [VBGAMER45/prettyurls#65]
```
Either change the pattern delimiters, or escape ~ characters.
```

Original issue reported on code.google.com by `curiousdannii` on 13 May 2009 at 2:48

**Comment by GoogleCodeExporter** [comment]
```
Fix for 1.0!
```

Original comment by `curiousdannii` on 13 May 2009 at 2:50
- Added labels: **Milestone-1.0**
- Removed labels: **Milestone-1.1**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in R98.
```

Original comment by `curiousdannii` on 24 May 2009 at 4:55
- Changed state: **Fixed**
